### DISCUSSION TITLE
Script error in cutscenes

### DISCUSSION THREAD

**Discussion by Prolesthefox** [JordanSantiagoYT/FNF-JS-Engine#760] [Q&A]
In the previous version this didn't happen to me, does anyone know why this happens?
![SED](https://github.com/user-attachments/assets/9c035d69-682f-4599-9643-1839f38b34fa)

**Comment by NAEL2XD** [comment]
missing mp4 file(?)

**Reply by Prolesthefox** [reply]
![Captura de pantalla 2024-11-26 223057](https://github.com/user-attachments/assets/75a2f309-433f-4774-8665-3b95ec7f79f9)
The video is in the corresponding folder and is .mp4

**Reply by NAEL2XD** [reply]
> ![Captura de pantalla 2024-11-26 223057](https://github.com/user-attachments/assets/75a2f309-433f-4774-8665-3b95ec7f79f9)
> The video is in the corresponding folder and is .mp4

Did you misspell the video name on your script that uses `startVideo`?
Example: `startVideo("hiimisspelledthisvideo")`

**Reply by Prolesthefox** [reply]
I wrote it well
![Captura de pantalla 2024-11-27 112456](https://github.com/user-attachments/assets/f21aee37-b2dd-444a-ac09-1fe214b39bf0)

**Reply by NAEL2XD** [reply]
try renaming the video that doesn't have spaces, don't forget the startVideo

**Reply by Prolesthefox** [reply]
Ok,Thx

**Reply by Prolesthefox** [reply]
It didn't work xd
![Captura de pantalla 2024-11-28 233706](https://github.com/user-attachments/assets/3b2870b6-df2d-4423-84ba-fb46936b2457)

**Reply by NAEL2XD** [reply]
Try setting codec to h264, this might fix it

**Reply by Prolesthefox** [reply]
I noticed that the glitch also happens in the cutscene event number 7
![Captura de pantalla 2024-11-29 224943](https://github.com/user-attachments/assets/c9f65034-2089-46f5-b27e-2975470e625d)
